roles/create_aws_backup_storage/tasks/main.yml

```diff
@@ -1,16 +1,25 @@
+- name: Get defaults
+  include_vars:
+    file: "{{ playbook_dir }}/config/defaults.yml"
+
 - name: Get AWS account information
   aws_caller_facts:
   register: caller_facts
 
 - name: Retrieve AWS account identifier (ARN) for use in resource tags
   set_fact:
     aws_arn: "{{ caller_facts.arn }}"
-    aws_resource_name: "velero-{{ caller_facts.arn | hash('md5') }}"
+    arn_region: "{{ caller_facts.arn }}-{{ awsRegion }}"
```

Comment:
Maybe we could rename to this `arn_and_region`? Using `arn_region` feels strange to me since the ARN of an account isn't attached to a region, although I'm pretty sure I know what you were going for with the name

Reply:
Might be a good idea to rename `aws_arn` to `aws_account_arn` while we're at it